### PR TITLE
Remember Me

### PR DESCRIPTION
**Pull request by mirulzaei**

### CODE REVIEW

**Comment by mirulzaei** [comment]
ready

File: config/settings.yml [hunk 1]
```diff
@@ -8,3 +8,10 @@ validate:
     min_length: 6
   post:
     max_length: 140
+<<<<<<< HEAD
```
Comment (manhnt-0960) [inline config/settings.yml]: let's fix conflict

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "All users") %>
+
+<h1>All users</h1>
+
+<%= paginate @users %>
+
+<ul class="users">
+    <%= render @users %>
```
Comment (manhnt-0960) [inline app/views/users/index.html.erb]: indent

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -7,6 +7,18 @@
...
+<<<<<<< HEAD
+
+       <%= f.label :remember_me, class: "checkbox inline" do %>
+        <%= f.check_box :remember_me %>
+        <span>Remember me on this computer</span>
+      <% end %>
+      
+=======
+>>>>>>> 77102e7... Login/logout
+>>>>>>> 5c450ee... Remember Me
```
Comment (manhnt-0960) [inline app/views/sessions/new.html.erb]: let's fix conflict

**Comment by mirulzaei** [comment]
ready

**Pull request merged**